sdk/how-to-guides/k8s-workload.md

```diff
@@ -503,15 +531,31 @@ As of Juju version 2.9.26, Pebble includes an HTTP `/v1/health` endpoint that al
 
 Each check can specify a `level` of "alive" or "ready". These have semantic meaning: "alive" means the check or the service it's connected to is up and running; "ready" means it's properly accepting network traffic. These correspond to Kubernetes ["liveness" and "readiness" probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/).
 
-When Juju creates a sidecar charm container, it initializes the Kubernetes liveness and readiness probes to hit the `/v1/health` endpoint with `?level=alive` and `?level=ready` filters, respectively.
+When Juju creates a sidecar charm container, it initialises the Kubernetes liveness and readiness probes to hit the `/v1/health` endpoint with `?level=alive` and `?level=ready` filters, respectively.
 
```

Comment:
This feels like something that we should document in Juju docs, but not sure where. Suggestions?

Reply:
Maybe it's just in reference, e.g. [docs/sdk/Pebble](https://juju.is/docs/sdk/pebble)?

Or if it's in a how-to it probably should be rephrased, right? So that it's saying something more like "to inform Kubernetes of the state of the container, ...".